src/settings/LocationLocations/locationDetail.test.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import user from '@testing-library/user-event';
+
+import LocationDetail from './LocationDetail';
+
+const mockInitialValues = {
+  id: '1',
+  servicePointIds: [],
+};
+
+jest.mock(
+  '@folio/stripes/core',
+  () => ({
+    stripesConnect: Component => props => <Component {...props} />,
+    IfPermission : ({ children }) => <>{children}</>
+  }),
+);
+
+jest.mock(
+  '@folio/stripes-components/lib/Icon',
+  () => {
+    return ({ children }) => <>{children}</>;
+  },
+);
+
+jest.mock(
+  './RemoteStorage',
+  () => ({
+    useRemoteStorageApi: () => ({ setMapping: jest.fn() }),
+  }),
+);
+
+const renderLocationDetail = ({
+  initialValues = mockInitialValues,
+  resources = {
+    institutions: {},
+    campuses: {},
+    libraries: {},
+  },
+  servicePointsById = {},
+  onEdit = jest.fn(),
+  onClone = jest.fn(),
+  onClose = jest.fn(),
+  onRemove = jest.fn(),
+  stripes = {
+    connect: jest.fn()
+  },
+} = {}) => (render(
+  <MemoryRouter>
+    <LocationDetail
+      initialValues={initialValues}
+      resources={resources}
+      servicePointsById={servicePointsById}
+      onEdit={onEdit}
+      onClone={onClone}
+      onClose={onClose}
+      onRemove={onRemove}
+      stripes={stripes}
+    />
+  </MemoryRouter>
+));
+
+describe('LocationDetail', () => {
+  it('should render all fields in location details', () => {
+    renderLocationDetail();
+
+    expect(screen.queryByText('ui-tenant-settings.settings.location.institutions.institution')).toBeVisible();
```

Comment:
```suggestion
    expect(screen.getByText(/institutions.institution/)).toBeVisible();
```

Reply:
`queryBy` is strongly recommended to use only for checking for non-existence.
See https://kentcdodds.com/blog/common-mistakes-with-react-testing-library/#using-query-variants-for-anything-except-checking-for-non-existence

And we can shorten the search criteria by using regex - this way we can drop identical part